src/registry/GithubRegistry.ts

```diff
@@ -162,6 +177,27 @@ export class GithubRegistry extends BaseRegistry implements IRegistry {
     throw new Error('TODO: Implement');
   }
 
+  public async getApiUrl(): Promise<string> {
+    const { remaining, reset } = await this.getApiRateLimit();
```

Comment:
I know we discussed maybe not adding this, but i thought this would be useful anyways with minimum code.

Ontop of the extra quota that is per user IP, this provides a slightly better UX by allowing users to call 
`hyperlane-dev warp read --symbol eth --verbosity debug --registry  https://github.com/ltyu/hyperlane-registry`  without `--disableProxy`, given they have enough quota.